charters/Architecture/charter.md

```diff
@@ -0,0 +1,35 @@
+# Architecture++
+
+## Description
+
+Architecture++ is responsible for the technical oversight of the Data Coordination Platform (DCP), including identifying and addressing *cross-cutting* architectural concerns across DCP components.
```

Comment:
There has been a slight preference to refer to DCP _projects_ rather than _components_.

Reply:
Slight preference from whom?  We have historically used the term "component" to refer to DCP subassemblies.  The term "Project" refers to a initiative or scope of work, a more vague term, which could mean "build a component" or "hold a meeting", whereas "component" implies a concrete deliverable.  The Architecture Team was created to address architectural issues that cut across components.  You may wish for it to also oversee other types of DCP projects, but this would be in addition to, not instead of, components.

Reply:
See the [extended conversation in the charter process PR](https://github.com/HumanCellAtlas/dcp-community/pull/1#discussion_r204456321) about products, projects, components, modules, etc.